src/test/regress/expected/multi_foreign_key.out

```diff
@@ -464,8 +462,7 @@ SELECT create_distributed_table('referencing_table', 'ref_id', 'hash');
 
 -- columns for the referenced table is empty
 ALTER TABLE referencing_table ADD CONSTRAINT test_constraint FOREIGN KEY(ref_id) REFERENCES referenced_table ON DELETE CASCADE;
-ERROR:  cannot create foreign key constraint because referenced column list is empty
-HINT:  Add column names to "REFERENCES" part of the statement.
+ERROR:  number of referencing and referenced columns for foreign key disagree
```

Comment:
Error message is generated by Postgres, i.e. user will be faced with the same error message running the same command without distributing tables.

`CREATE TABLE referenced_table(id int UNIQUE, test_column int, PRIMARY KEY(id, test_column));
CREATE TABLE referencing_table(id int, ref_id int);
ALTER TABLE referencing_table ADD CONSTRAINT test_constraint FOREIGN KEY(ref_id) REFERENCES referenced_table ON DELETE CASCADE;
ERROR:  number of referencing and referenced columns for foreign key disagree`